### PR TITLE
Second attempt at early allocation of CWT Labels

### PR DESCRIPTION
**Pull request by laurencelundblade**
This brings in the CWT Labels for EAT claims including updating the CDDL and examples.

This also fill out all the IANA section for the TBD items too, that was previously a "TODO" item.

### CODE REVIEW

**Comment by laurencelundblade** [comment]
This addresses #131

File: draft-ietf-rats-eat.md [hunk 1]
```diff
@@ -1827,28 +1832,58 @@ The JWT Claim Names and CWT Claim Keys are not expected to change.
 * Claim Name: UEID
 * Claim Description: The Universal Entity ID
 * JWT Claim Name: "ueid"
-* CWT Claim Key: 11
+* CWT Claim Key: 256
```
Comment (selfissued) [inline draft-ietf-rats-eat.md:1835]: Please update all the "CWT Claim Key" values to use the form:

- CWT Claim Key: TBD (requested value NNN)

For instance, this example should be:

- CWT Claim Key: TBD (requested value 256)

File: draft-ietf-rats-eat.md [hunk 2]
```diff
@@ -2029,7 +2029,7 @@ This early allocation will presumably complete correctly
 * Claim Name: SW Manifests
 * Claim Description: Manifests describing the SW installed on the entity
 * JWT Claim Name: "manifests"
-* Claim Key: TBD273
+* Claim Key: TBD
```
Comment (henkbirkholz) [inline draft-ietf-rats-eat.md:2032]: The CDDL defintion for this Claim includes "manifests-type = [+ $$manifest-formats]", which means that you can express multiple CoSWIDs or SUIT Manifests as content of an EAT. As CoSWIDs are either "Evidence" (which literally translates to Claims produced by an Attester) or "Payload" ("which means they are Endorsements that can be co-located with endorsed software components on an Attester), why should they be produced by the Verifier and included in Attestation Results? See below, the output of an appraisal conducted by a Verifier that is "swresult".

File: draft-ietf-rats-eat.md [hunk 3]
```diff
@@ -2039,7 +2039,7 @@ This early allocation will presumably complete correctly
 * Claim Name: SW Evidence
 * Claim Description: Measurements of the SW, memory configuration and such on the entity
 * JWT Claim Name: "swevidence"
-* Claim Key: TBD274
+* Claim Key: TBD
```
Comment (henkbirkholz) [inline draft-ietf-rats-eat.md:2042]: Why should a Verifier be allowed to include this Claim in Attestation Results. The descriptive text clearly implies that this Claim must originate from an Attester. What is the purpose of allowing this for Attestation Results?

File: draft-ietf-rats-eat.md [hunk 4]
```diff
@@ -2049,7 +2049,7 @@ This early allocation will presumably complete correctly
 * Claim Name: SW Measurment Results
 * Claim Description: The results of comparing SW measurements to reference values
 * JWT Claim Name: "swresults"
-* Claim Key: TBD275
+* Claim Key: TBD
```
Comment (henkbirkholz) [inline draft-ietf-rats-eat.md:2052]: The complement to "swevidence". Why should this Claim ever be produced by an Attester? What is the purpose of allowing this for Evidence?

**Review by henkbirkholz — CHANGES_REQUESTED** [review]
TL;DR

The axiom/principle-decision to allow for every Claim defined by EAT to be included in both RATS Conceptual Messages that are Evidence and Attestation Results appears to be quite confusing to me.

A more detailed overview and selected examples are included in the comments for [draft-ietf-rats-eat.md](https://github.com/ietf-rats-wg/eat/pull/152/files/c8b39f500de29237c1b3916bb8249c27c7ee2fdd..de7f7230764985b30fdf6e728655b4736520f02b#diff-a14dc11088917fcc59b3a1cf2f73b0fd477930cf32583c0264038cd78e68fd9d)
